### DISCUSSION TITLE
What control should I use for Text Area, i.e. multiline plain text input

### DISCUSSION THREAD

**Discussion by quintusm** [Megabit/Blazorise#2630] [Q&A]
I Use TextEdit for single line input, but don't see a way to show a multiline area for text input.

**Comment by stsrki** [comment]
For multiline text input you can use MemoEdit component.

**Reply by quintusm** [reply]
Ah, I see.  

Sorry, somehow I missed it.

Thank you very much for prompt response and hard work on this toolkit
